hw/ip/flash_ctrl/doc/checklist.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "FLASH_CTRL Checklist"
+---
+
+This checklist is for [Hardware Stage]({{< relref "/doc/project/hw_stages.md" >}}) transitions for the [FLASH_CTRL peripheral.]({{< relref "index.md" >}})
+All checklist items refer to the content in the [Checklist.]({{< relref "/doc/project/checklist.md" >}})
+
+## Design Checklist
+
+### D1
+
+Type          | Item                  | Resolution  | Note/Collaterals
+--------------|-----------------------|-------------|------------------
+Documentation | [SPEC_COMPLETE][]     | Done        | [FLASH_CTRL Spec][]
+Documentation | [CSR_DEFINED][]       | Done        |
+RTL           | [CLKRST_CONNECTED][]  | Done        |
+RTL           | [IP_TOP][]            | Done        |
+RTL           | [IP_INSTANCED][]      | Done        |
```

Comment:
Is this still on the checklist? I thought we wanted to remove the "instanced in top-level" since D1 implies instantiability.

Reply:
yeah, we did. I guess we should remove it from the template first?